src/main/resources/resources/logisim/strings/gui/gui_pl.properties

```diff
@@ -685,16 +687,20 @@ argTemplateOptionArgName = plik
 # ==> invalidLocaleError =
 # ==> invalidLocaleOptionsHeader =
 loadMultipleError = Opcję "--load" można podać tylko raz.
-loadNeedsTtyError = Opcja "--load" działa tylko w połączeniu z opcją "-tty".
+loadNeedsTtyError = Opcja "--load" działa tylko w połączeniu z opcją "--tty".
+# ==> saveMultipleError =
+# ==> saveNeedsTtyError = 
```

Comment:
Would be

    saveNeedsTtyError = Opcja "--save" działa tylko w połączeniu z opcją "--tty".